modules/cp4ba/variables.tf

```diff
@@ -1,61 +1,84 @@
-variable "enable" {
-  default     = true
-  description = "If set to true installs Cloud-Pak for Business Automation on the given cluster"
+variable "ibmcloud_api_key" {
+  description = "Enter your IBM API Cloud access key. Visit this link for more information: https://cloud.ibm.com/docs/account?topic=account-userapikey&interface=ui "
 }
 
-variable "cluster_name_or_id" {
+variable "cluster_id" {
   default     = ""
-  description = "Enter your cluster id or name to install the Cloud Pak. Leave blank to provision a new Openshift cluster."
+  description = "Set your cluster ID to install the Cloud Pak for Business Automation. Leave blank to provision a new OpenShift cluster."
 }
```

Comment:
This can not be left blank here in the module.  Needs to be required.  So no default either.

Reply:
I removed the: `default     = ""`